### PR TITLE
Update manage-rdp-device-redirections.md

### PR DESCRIPTION
**Pull request by kmsft**
Updated with Nov 2021 version (1 year newer) of the spreadsheet

### CODE REVIEW

**Review by kmsft — COMMENTED** [review]
added win11 links

File: windows-365/enterprise/manage-rdp-device-redirections.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ For more help using the settings catalog to create a device configuration policy
 
 ## Use a GPO to manage RDP device redirections
 
-To manage any of the redirections by using GPO, create and assign a GPO in your Windows Server Active Directory domain. Make sure to use the corresponding policies as shown in the [RDP device redirection settings table](#rdp-device-redirection-settings). To learn more about the policies, download the [Group Policy Settings Reference Spreadsheet](https://www.microsoft.com/download/101451).
+To manage any of the redirections by using GPO, create and assign a GPO in your Windows Server Active Directory domain. Make sure to use the corresponding policies as shown in the [RDP device redirection settings table](#rdp-device-redirection-settings). To learn more about the policies, download the [Group Policy Settings Reference Spreadsheet for Windows 10 November 2021 Update [21H2]](https://www.microsoft.com/download/103668) and [Group Policy Settings Reference Spreadsheet for Windows 11 2022 [22H2]]https://www.microsoft.com/download/104594).
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/manage-rdp-device-redirections.md:93]:
```suggestion
To manage any of the redirections by using GPO, create and assign a GPO in your Windows Server Active Directory domain. Make sure to use the corresponding policies as shown in the [RDP device redirection settings table](#rdp-device-redirection-settings). To learn more about the policies, download the [Group Policy Settings Reference Spreadsheet for Windows 10 November 2021 Update [21H2]](https://www.microsoft.com/download/103668) and [Group Policy Settings Reference Spreadsheet for Windows 11 2022 [22H2]](https://www.microsoft.com/download/104594).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft , kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
@kmsft Please recreate this PR in the private repo. Closing.

**Pull request closed without merging**